src/pages/ReportParticipantsPage.js

```diff
@@ -57,25 +57,27 @@ const defaultProps = {
 const getAllParticipants = (report, personalDetails) => {
     const {participants} = report;
 
-    return _.map(participants, (login) => {
-        const userLogin = Str.removeSMSDomain(login);
-        const userPersonalDetail = lodashGet(personalDetails, login, {displayName: userLogin, avatar: ''});
+    return _.chain()
```

Comment:
Uhh, don't we have to chain the participants 😄

Reply:
why not?

Reply:
Right now it's chaining nothing. I think it should be `_.chain(participants)` right?

Reply:
ahhh you right you right sorry I'm working too fast